installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -0,0 +1,20 @@
+[id="installing-sno"]
+= Installing single node OpenShift
+:context: install-sno-installing-sno
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[IMPORTANT]
+====
+Single node OpenShift is a Developer Preview feature only. Developer Preview features
```

Comment:
"Installing {product-title} on a single node is a Developer Preview feature only."